app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] @team kindly ensure your login/registration pages match the wireframe design given below is a comparison for  better understanding :


Design Given          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/216150334-c518966d-112c-4fd8-9394-d368c1ee770a.png) |  ![image](https://user-images.githubusercontent.com/61408860/216150402-58ad092e-19e2-4810-9970-8dfafa075df1.png)

Reply:
Hello @danmainah, for this step should we remove the sign-up, remember me and forgot your password? or we should only add a username instead of an email? Thank you! 🙌🏽

Reply:
- Well resolved @OscarFMdev